strings.txt

```diff
@@ -20854,3 +20854,7 @@
   [traffic_update_app_message]
     en = To display traffic data, the application must be updated.
     ru = Для отображения пробок необходимо обновить приложение.
+
+  [traffic_data_unavailable]
```

Comment:
https://books.google.com/ngrams/graph?content=data+is+unavailable%2C+data+is+not+available&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cdata%20is%20unavailable%3B%2Cc0%3B.t1%3B%2Cdata%20is%20not%20available%3B%2Cc0